target-platform/target-platform.target

```diff
@@ -97,6 +97,12 @@
 					<version>31.1-jre</version>
 					<type>jar</type>
 				</dependency>
+				<dependency>
+					<groupId>commons-io</groupId>
+					<artifactId>commons-io</artifactId>
+					<version>2.11.0</version>
+					<type>jar</type>
+				</dependency>
```

Comment:
Is this necessary for this change?

Reply:
See second bullet point. Recommended replacement for some of the Maven FileUtils methods is Apache Commons FileUtils (according to their Javadoc).

Reply:
Acknowledge.
But it looks like commons-io is already in the TP contributed by some IU-loctions so we don't need that extra dependency.